src/convertToObject.js

```diff
@@ -12,7 +13,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
checklist p.2 [CODE STYLE] - Avoid variable namings like array , object arr, obj, result. It will decrease code readability

Reply:
fixed